### ISSUE TITLE
Did it stop working?

### ISSUE THREAD

**Issue by gaveitatry02** [aligo/better-google#3]
Is it just my imagination or did the Better Google userscript stop working?  I didn't notice it broken yesterday, but today all the Google search results appear like I don't want (without the userscript changes).

**Comment by tejaslodaya** [comment]
Yes, +1

**Comment by drwonky** [comment]
Google changed the "r" class to "yuRUbf" in the result heading.

By changing references to ".r" in the script to ".yuRUbf" it fixes the layout.

**Comment by tejaslodaya** [comment]
Thanks @drwonky !

Attaching script - 
```
// ==UserScript==
// @name         Better Google
// @namespace    google
// @version      0.1.14
// @description  Don't be evil
// @author       aligo, adambh
// @license      MIT
// @supportURL   https://github.com/aligo/better-google
// @match        https://*.google.com/search?*
// @include      /^https?://(?:www|encrypted|ipv[46])\.google\.[^/]+/(?:$|[#?]|search|webhp)/
// @grant        none
// @run-at       document-start
// ==/UserScript==

(function() {
    'use strict';

    var betterGoogleRow = function(el) {
        var tbwUpd = el.querySelectorAll('.TbwUpd');
        if (tbwUpd.length > 0) {
            var linkEl = el.querySelector('.yuRUbf > a');
            var addEl = linkEl.nextSibling;

            var betterAddEl = document.createElement('div');
            betterAddEl.className = 'btrAdd';

            if (addEl) {
                for (var i = 0; i < addEl.children.length; i++) {
                    var _el = addEl.children[i];
                    if (_el.className.indexOf('TbwUpd') == -1) {
                        betterAddEl.appendChild(_el);
                    }
                }
            }

            var betterEl = document.createElement('div');
            betterEl.className = 'btrG';
            betterEl.appendChild(betterAddEl);

            el.querySelector('.yuRUbf').appendChild(betterEl);

            var urlEl = document.createElement('a');
            urlEl.href = linkEl.href;
            urlEl.target = '_blank';
            urlEl.className = 'btrLink';

            var urlCiteEl = document.createElement('cite');
            urlCiteEl.innerText = linkEl.href;
            urlCiteEl.className = 'iUh30 bc';
            urlEl.appendChild(urlCiteEl);


            var maxWidth = el.clientWidth - betterAddEl.offsetWidth - 10;

            betterEl.insertBefore(urlEl, betterAddEl);
            if (urlEl.offsetWidth > maxWidth) {
                urlEl.style.width = maxWidth.toString() + 'px';
            }


            tbwUpd.forEach(function(el) { el.remove() });
            linkEl.querySelector('br:first-child').remove();
        }
    }

    var prevResultCount = 0;
    var bettered = false;

    var runBetterGoogle = function() {
        if (prevResultCount != document.querySelectorAll('.g .rc').length) {
            document.querySelectorAll('.g .rc').forEach(betterGoogleRow);
            prevResultCount = document.querySelectorAll('.g .rc').length;
        }
        if ( !bettered ) {
            if ( MutationObserver != undefined ) {
                var searchEl = document.getElementById('rcnt');
                var observer = new MutationObserver(runBetterGoogle);
                observer.observe(searchEl, {childList: true, subtree: true});
            }
            bettered = true;
        }
    };

    var prepareStyleSheet = function() {
        var style = document.createElement('style');
        style.setAttribute('media', 'screen');
        style.appendChild(document.createTextNode(''));
        document.head.appendChild(style);
        style.sheet.insertRule('.btrG { word-break: break-all; line-height: 18px; }');
        style.sheet.insertRule('.btrG .btrAdd { display: inline-block; vertical-align: top; }');
        style.sheet.insertRule('.btrG .btrLink { display: inline-block; vertical-align: top; line-height: 18px; overflow: hidden; text-overflow: ellipsis; white-space: nowrap; text-decoration: none !important; }');
        style.sheet.insertRule('.btrG .btrLink cite.iUh30 { color: #006621; font-size: 16px; }');
    };

    var checkElementThenRun = function(selector, func) {
        var el = document.querySelector(selector);
        if ( el == null ) {
            if (window.requestAnimationFrame != undefined) {
                window.requestAnimationFrame(function(){ checkElementThenRun(selector, func)});
            } else {
                document.addEventListener('readystatechange', function(e) {
                    if (document.readyState == 'complete') {
                        func();
                    }
                });
            }
        } else {
            func();
        }
    }

    checkElementThenRun('head', prepareStyleSheet);
    checkElementThenRun('#rcnt', runBetterGoogle);
})();
```

**Comment by tejaslodaya** [comment]
@aligo can you merge the PR?

**Comment by volcano1111** [comment]
> 
> 
> Google changed the "r" class to "yuRUbf" in the result heading.
> 
> By changing references to ".r" in the script to ".yuRUbf" it fixes the layout.

Didn't work for me, I don't know why :(

**Comment by drwonky** [comment]
There are 2 references to the exact string __.r__ in the script, both must be replaced with __.yuRUbf__

**Comment by Perty65** [comment]
Please where do I copy and paste the new script provided? I mean where can I find the script script on crome. I just see tampermoney. I cant live without this script.

**Comment by gaveitatry02** [comment]
@Perty65 - If you can't figure it out (I haven't even tried yet), you can install this user script (https://greasyfork.org/en/scripts/411886-better-google-updated).  Basically someone saw that the Better Google script was broken, made the necessary fixes, and then republished it with a message "This will be removed once the original creator updates the script themselves."  So I"m thinking you and me can use his script until the original one gets patched.

**Comment by gaveitatry02** [comment]
For those who are trying to modify the original script themselves, don't search for ".r" because you will come up with things like "el.remove" which contains ".r".
I would search for this:
el.querySelector('.r
And then you will get exactly two search results and then you can replace the ".r" portion with ".yuRUbf".  I haven't tried it myself though, because I am using e11ding's temporary script and I want to see if the original script will get an update by itself.

**Comment by aligo** [comment]
Solved by #5. Updated on greasyfork. Sorry for or the late response.

Thank you all.
